holidays/countries/micronesia.py

```
@@ -0,0 +1,205 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)